src/main/java/camp/nextstep/edu/level1/lotto/lotto/LottoNumbers.java

```diff
@@ -0,0 +1,77 @@
+package camp.nextstep.edu.level1.lotto.lotto;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class LottoNumbers {
+    private static final int LOTTO_START_NUMBER = 1;
+    private static final int LOTTO_END_NUMBER = 45;
+    private static final int LOTTO_RANGE = 6;
+    private static final String PRINT_JOIN_DELIMITER = ", ";
+    private static final String TO_STRING_PREFIX = "[";
+    private static final String TO_STRING_SUFFIX = "]";
+    private static final List<Integer> lottoNumberPreset = new ArrayList<>();
+
+    static {
+        for (int number = LOTTO_START_NUMBER; number <= LOTTO_END_NUMBER; number++) {
+            lottoNumberPreset.add(number);
+        }
+    }
+
+    private final Set<LottoNumber> lottoNumbers = new HashSet<>();
+
+    public LottoNumbers() {
+        this(createRandomLottoNumbers());
+    }
+
+    public LottoNumbers(Collection<Integer> numbers) {
+        checkLottoNumbers(numbers);
+
+        addAll(numbers);
+    }
+
+    public long matchedCountByWinnerNumbers(LottoNumbers winnerNumbers) {
+        return winnerNumbers.lottoNumbers.stream().filter(this::hasContainLottoNumber).count();
```

Comment:
```suggestion
        return winnerNumbers.lottoNumbers.stream()
        .filter(this::hasContainLottoNumber)
        .count();
```
스트림은 행위 단위로 내려쓸 경우 가독성과 디버깅에 장점이 많을 것같아요.

Reply:
넵. 감사합니다.